### DISCUSSION TITLE
Add the option to remove or hide "Shred with BleachBit" in the Windows Context Menu

### DISCUSSION THREAD

**Discussion by ppw0** [bleachbit/bleachbit#1715] [Ideas]
Hi,

would it be possible to add an option to remove the "Shred with BleachBit" entry from the Windows context menu? I had to manually find and remove the relevant registry entries because I could not find that option in the Preferences section - the only one related had a disclaimer that it only applies to KDE.

**Comment by az0** [comment]
![BleachBit_Screenshot8](https://github.com/user-attachments/assets/1e2bdac8-6346-4d22-8359-b01a44d635b6)

There is an option during installation on this screen, though it's cut off in this screenshot. Just scroll down. Is this helpful?

**Reply by ppw0** [reply]
Hi Andrew, thanks for replying. It seems that using a package manager like Chocolatey can limit configuration options during installation. Is there a way to pass additional installation options to the installer? If so, I could try forcing Chocolatey to install the package using them.

**Comment by az0** [comment]
There are a few installation switches in the docs https://docs.bleachbit.org/doc/install-on-windows.html

However there is not a switch for that.

In short term, what do you think about deleting one registry key?
HKCR AllFilesystemObjects\shell\shred.bleachbit

**Reply by ppw0** [reply]
It's what I did. (See original post)

**Reply by Albirew** [reply]
for the next people, until a way to add/remove it from installation switch and/or settings page appears:
```reg
Windows Registry Editor Version 5.00

[-HKEY_CLASSES_ROOT\AllFileSystemObjects\shell\shred.bleachbit]
```
